intermittent-failure-tracker/map.jinja

```diff
@@ -1,5 +1,5 @@
 {%
   set tracker = {
-    'rev': '37ef73bb0059e0555a903e758dae8006c46a9916'
+    'rev': 'f2f3825b6b026b54769a995ae8a526b2d81ee5fd'
```

Comment:
It would be nice to publish all these packages to PyPI as well, as our Salt version does better with that than git installs.

Reply:
Better in what sense?

Reply:
https://github.com/servo/saltfs/issues/593 is what I'm aware of; this may have been fixed or improved since then but at least in our old Salt the git installs seem less well supported.

Reply:
Per my latest comment in that issue, a2934d015b300086b1f6194f5b69d785ebaf4df0 is all that is necessary. I would rather not go through the effort of publishing these packages.

Reply:
Thanks for updating that issue, since it seems to be working then git dependencies are fine by me.